<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reader_card</class>
 <widget class="QDialog" name="reader_card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>537</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Card reader</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Surname:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_fam"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Name:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_ima"/>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Patronymic:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_otc"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Date of birth:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit_date_r">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Ticket number:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_num"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Address:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_address"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Phone:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_phone"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Document:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_doc"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QPushButton" name="pushButton_del">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Del</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_toFirst">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>&lt;&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_toPrev">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_toNext">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_toLast">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>&gt;&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_save">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_close">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Books</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTableWidget" name="tableWidget_active">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>27</number>
             </attribute>
             <column>
              <property name="text">
               <string>id</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Identifier</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>ISBN</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Title of book</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Author</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date of receipt</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QPushButton" name="pushButton_add_book">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Add book</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>History</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget_history">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>27</number>
           </attribute>
           <column>
            <property name="text">
             <string>id</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Identifier</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ISBN</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Title of book</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Author</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Date of receipt</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Date of return</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_status">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
